src/Collapse.js

```diff
@@ -81,14 +78,15 @@ const Collapse = React.createClass({
 
 
   getMotionHeight(height) {
-    const {isOpened, springConfig} = this.props;
+    const {isOpened, springConfig, fixedHeight} = this.props;
```

Comment:
`getMotionHeight` is only ever called when `fixedHeight > -1`, so it does not seem to make sense to have it here